.editorconfig

```diff
@@ -0,0 +1,766 @@
+[*]
+charset = utf-8
```

Comment:
I need to look into this editorconfig stuff. Would probably be best to do it in a separate PR. This file also looks like it has way too much stuff in it. I would want to trim out anything we don't need.